plugins/network/multibandwidth

```diff
@@ -101,11 +104,20 @@ for host in $hosts; do
                 | cut -d "b" -f1)
 
         if (echo "$SPEED" | grep -q "M"); then
-                echo "$SPEED" | awk '{a+=$1} END{print a*1000000}'
+                VALUE=`echo "$SPEED" | sed 's/.$//'`
+                RATE=`echo "$VALUE * 1048576" | bc -l`
+
+                if [ `echo "$RATE > $max_mbps" | bc` -eq "1" ]; then
```

Comment:
Are you sure, that you need to introduce the dependency on `bc`?
How about the following (even for non-integer values):
```
if [ "${RATE%.*}" -ge "$max_mbps" ]; then
```